src/runtime/tests/storage-proxy-test.ts

```diff
@@ -807,4 +811,28 @@ describe('storage-proxy', () => {
     await engine.verifySubsequence('onHandleUpdate:P2:bar:+[v2](originator)');
     await engine.verify('HandleStore:bar:v2');
   });
+
+  it('makes sure that noOpStorageProxy class contains all the functions implemented in the other storageProxies', async () => {
```

Comment:
Maybe make it shorter? 
"ensures NoOpStorageProxy overrides all methods"

FWIW `it` here refers to the class under test, so here it is StorageProxy (I reckon). Ideally the sentence would read with it == Storage Proxy.

As above:
it (Storage Proxy) delivers the originator status to the originating particle

But I'm just saying it for your next parches. Here we cannot make the message properly, unless we would write a test for a NoOpStorageProxy, and then we should do:

`it('overrides all methods of the StorageProxy',` ...

But well, it's ok.

Reply:
Got it! Changed the test name! :)